README.md

```diff
@@ -95,7 +107,12 @@ This is an example `TenorImage` object:
   tags: [ "American Psycho", "Patrick Bateman", "American", "psycho"],
   url: "https://media.tenor.com/5lLcKZgmIhgAAAAC/american-psycho-patrick-bateman.gif",
   height: 240,
-  width: 244
+  width: 244,
+  preview: {
+    url: "https://media.tenor.com/5lLcKZgmIhgAAAAC/american-psycho-patrick-bateman.gif",
```

Comment:
`https://media.tenor.com/5lLcKZgmIhgAAAAM/american-psycho-patrick-bateman.gif`

Reply:
Wow that ONE letter difference I never noticed 😅 makes a lot more sense now haha